account_invoice_change_currency/models/account_move_line.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2017-2018 Vauxoo
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    base_price_unit = fields.Monetary(
```

Comment:
I think base is not clear enough, as it could be confused with the base price (price without taxes)

Reply:
Done!